kolibri/core/webpack/management/commands/webpack_json.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+
+from kolibri.core.webpack.hooks import WebpackBundleHook
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    args = ()
+    help = 'Creates a new schema'  # @ReservedAssignment
+    option_list = BaseCommand.option_list + ()
+
+    def handle(self, *args, **options):
+
+        logging.debug(args)
+
+        for hook in WebpackBundleHook().registered_hooks:
```

Comment:
Essentially, this iterates over all enabled plugins, yes?

I had been thinking of suggesting this modification, so am glad to see that you had made it here. As such, I think we may be able to simplify the webpack config code even more. I will pull in your branch and suggest a PR.